glommio/src/executor/mod.rs

```diff
@@ -2338,6 +2340,69 @@ impl ExecutorProxy {
                 .map(|x| ScopedTask::<'a, T>(x, PhantomData))
         })
     }
+
+    /// Spawns a blocking task into a background thread where blocking is
+    /// acceptable.
+    ///
+    /// Glommio depends on cooperation from tasks in order to drive IO and meet
+    /// latency requirements. Unyielding tasks are detrimental to the
+    /// performance of the overall system, not just to the performance of
+    /// the one stalling task.
+    ///
+    /// `spawn_blocking` is there as a last resort when a blocking task needs to
+    /// be executed and cannot be made cooperative. Examples are:
+    /// * Expensive syscalls that cannot use `io_uring`, such as `mmap`
+    ///   (especially with `MAP_POPULATE`)
+    /// * Calls to synchronous third-party code (compression, encoding, etc.)
+    ///
+    /// It is recommended to guard against calling this method if the blocking
```

Comment:
I don't agree, and we shouldn't be in the business of making this recommendation.

Let's remove this text

And if you're curious I can give you a lot of examples where even seemingly harmless things like "small mmap" can take 7+ seconds.